ColorShapeSorting/ColorShapeSortingJSONSchema.json

```diff
@@ -0,0 +1,502 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "shapeColor_taskData",
+  "title": "JSON Schema for the Shape-Color Sorting MTB Test.",
+  "description": "This schema shares many elements across all Mobile Toolbox tests, but it is specific to the Shape-Color Sorting task. Common elements across tasks appear in the definitions section (though not all of them are utilized herein), but the main properties section defines the task-specific representations of it.",
+  "type": "object",
+  "definitions":{
+    "specialStepsIndicators": {
+      "type": "boolean",
+      "default": false,
+      "description": "This definition will be reused for instructions, practice, interruptions, etc., as needed and described in the element properties, below."
+   },
+   "timemarks":{
+     "type": "string",
+     "format": "date-time",
+     "description": "The timestamp when an event occurred, following ISO 8601 formatting conventions"
+   },
+   "taskRunUUID":{
+     "type": "string",
+     "format": "uuid",
+     "description": "This is a global identifier"
+   },
+   "taskStatus":{
+     "type": "string",
+     "description": "completion status for the task."
+     "enum":: ["complete","willNotComplete","partiallyComplete"]
```

Comment:
I think that there might be a double colon here instead of a single one.

Reply:
The Shape-Color Sorting schema is still a work-in-progress. Its not meant to be used yet. But yes, we need to take away the double colon. Do you want that fixed before merging or not since it is not implemented yet?

And yes, we described this change as "opening the floodgates" - it is intentional to enable additional properties at all levels.